silnlp/common/paratext.py

```diff
@@ -164,7 +164,8 @@ def extract_terms_list(list_type: str, project: Optional[str] = None) -> Dict[st
         "w", encoding="utf-8", newline="\n"
     ) as terms_glosses_file, terms_vrefs_path.open("w", encoding="utf-8", newline="\n") as terms_vrefs_file:
         if os.path.exists(terms_xml_path):
-            terms_tree = etree.parse(str(terms_xml_path))
+            with terms_xml_path.open("rb") as termsf:
```

Comment:
Can you change the variable names for open files to something like `terms_file`?

Reply:
done.  All termsf and settingsf, etc. updated to terms_file etc.